### ISSUE TITLE
Could you provide the weights file?

### ISSUE THREAD

**Issue by 22109095** [YangLiu14/Open-World-Tracking#1]
Hello, after reading this excellent paper, I want to try to run the code, could you provide the weights file? Thank you very much!

**Comment by YangLiu14** [comment]
Hi, thank you for your message. We were using a model from detectron2 to generate propopsals per frame. This is the [link](https://github.com/YangLiu14/detectron2-TAO) of that forked-repo. Sadly this was an old version of detectron2, so it only supports until torch1.5.  (I'm trying to adapt the current version of detectron2, I'm update the repo as soon as I get it done.)

This is the model **config** I used 
https://github.com/YangLiu14/detectron2-TAO/blob/master/configs/Misc/noNMS/panoptic_fpn_R_101_dconv_cascade_gn_3x.yaml

The model **weights** is avaiable in the official detectron2 repo: https://dl.fbaipublicfiles.com/detectron2/Misc/panoptic_fpn_R_101_dconv_cascade_gn_3x/139797668/model_final_be35db.pkl

Detectron2 now provides new models that have better performances. You are encouraged to try them out. Just modify a few places to output `background scores`, `objectness scores` and `logits from FC layer` (and other stuffs we mentioned in the "Proposal Generation" section), it is expected to have slightly better proposals than the model mentioned above.

**Comment by 22109095** [comment]
Thank you for the thoughtful reply,  I wish you a happy life, best wishes!
